src/Client.php

```diff
@@ -172,16 +172,17 @@ public function getEndpoint(): string
     public function getUserAgent(): string
     {
         if (null !== $this->userAgentPrefix) {
+            // Checks if userAgent has multiple values in string.
             $userAgent = explode('~', $this->userAgentPrefix);
             if (count($userAgent) > 1) {
                 if ('' === $userAgent[1]) {
-                    return sprintf("{$userAgent[0]} ({$this->getClientVersion()}; {$userAgent[2]}; {$this->getPHPVersion()})");
+                    return "{$userAgent[0]} ({$this->getClientVersion()}; {$userAgent[2]}; PHP/" . phpversion() . ")";
                 }
 
-                return sprintf("{$userAgent[1]} ({$userAgent[0]}; {$this->getClientVersion()}; {$userAgent[2]}; {$this->getPHPVersion()})");
+                return "{$userAgent[1]} ({$userAgent[0]}; {$this->getClientVersion()}; {$userAgent[2]}; PHP/" . phpversion() . ")";
```

Comment:
Use `PHP_VERSION` instead.

https://github.com/search?l=PHP&q=PHP_VERSION&type=Code
https://github.com/search?q=%22phpversion%28%29%22+language%3APHP&type=Code&ref=advsearch&l=PHP&l=

https://www.php.net/manual/en/reserved.constants.php#reserved.constants.core

Reply:
Thanks @mxr576 ,
Yes, missed `PHP_VERSION`.
Also removed the complex user-agent string code and instead will send it from dependent modules
according to this comment https://github.com/apigee/apigee-edge-drupal/issues/556#issuecomment-894076924